spyderlib/spyder.py

```diff
@@ -1541,14 +1489,14 @@ def setup_default_layouts(self, index, settings):
                       finder] + plugins,
                      [console_int, console_ext, console_ipy, history]]
                     ],
-                    'width fraction': [0.0,
+                    'width fraction': [0.0,             # column 0 width
                                        0.55,            # column 1 width
-                                       0.0,
+                                       0.0,             # column 2 width
                                        0.45],           # column 3 width
                     'height fraction': [[1.0],          # column 0, row heights
-                                        [1.0],   # column 1, row heights
-                                        [0.55, 0.45],          # column 2, row heights
-                                        [1.0]],   # column 0, row heights
+                                        [1.0],          # column 1, row heights
+                                        [1.0],          # column 2, row heights
+                                        [0.55, 0.45]],  # column 3, row heights

```

Comment:
Still overflowing, but that's related to the timer you're using above, not to this.